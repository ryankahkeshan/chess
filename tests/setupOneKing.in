setup
+ K e1
done
+ K e2
done
+ k e8
done
- e2
done
