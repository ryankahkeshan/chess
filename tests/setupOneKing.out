8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

Invalid Setup: 
  - Each player must have exactly one king
  - There should be no pawns on the first or last row
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _K_ _
1 _ _ K _ 

  abcdefgh

Invalid Setup: 
  - Each player must have exactly one king
  - There should be no pawns on the first or last row
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _K_ _
1 _ _ K _ 

  abcdefgh

Invalid Setup: 
  - Each player must have exactly one king
  - There should be no pawns on the first or last row
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

Final Score:
White: 0
Black: 0
